modules/core/src/core/lib/layer.js

```diff
@@ -362,6 +362,71 @@ export default class Layer {
     }
   }
 
+  // This method overwrites the current buffer and is used for multi-picking
+  _clearInstancePickingColor(color) {
+    const {instancePickingColors} = this.getAttributeManager().attributes;
+    const {state: attribute} = instancePickingColors;
+    const {value, size} = attribute;
+
+    const i = this.decodePickingColor(color);
+    value[i * size + 0] = 0;
```

Comment:
I think there is a `this.nullPickingColor()` method.

Reply:
Yes, but it doesn't really help me much.